Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -27,6 +27,9 @@
         prob: 0.3
       - id: WeaponEnergyGunMini #LPP
       - id: LPPQMPanties #LPP
+      - id: LPPRubberStampConfidentional #LPP
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)